components/licensor/typescript/ee/main.go

```diff
@@ -7,22 +7,27 @@ package main
 import (
 	"C"
 	"encoding/json"
+	"os"
 
 	log "github.com/sirupsen/logrus"
 
 	"github.com/gitpod-io/gitpod/licensor/ee/pkg/licensor"
 )
 
 var (
-	instances map[int]*licensor.Evaluator = make(map[int]*licensor.Evaluator)
-	nextID    int                         = 1
+	instances map[int]licensor.Evaluator = make(map[int]licensor.Evaluator)
+	nextID    int                       = 1
 )
 
 // Init initializes the global license evaluator from an environment variable
 //export Init
 func Init(key *C.char, domain *C.char) (id int) {
 	id = nextID
-	instances[id] = licensor.NewEvaluator([]byte(C.GoString(key)), C.GoString(domain))
+	if os.Getenv("use_replicated") == "true" {
```

Comment:
Considering this env var will be part of `server` (and all other services which might integrate with `licensor` down the road), let's think of a more apt env var name :)

E.g. `GITPOD_LICENSE_TYPE` which can be `replicated` or anything else in which case we'd fall back to the `GitpodEvaluator`